src/main/java/pl/ug/virtualofficebackend/domain/entity/DecorationEntity.java

```diff
@@ -0,0 +1,129 @@
+package pl.ug.virtualofficebackend.domain.entity;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "decoration", schema = "public", catalog = "postgres")
+public class DecorationEntity {

```

Comment:
Entity?
a bedziemy miec coś takiego jak zwykłe np. Decoration? jeśli tak, to co to bedzie oznaczać?
bo imho klasa Decoration (sama nazwa) wystarczy, aby pokazać, że jest to tabela

Reply:
Docelowo nie chce działać na modelach z bazy, dlatego nazwałem je z suffixem Entity. imo zazwyczaj jest w nich sporo useless pól i możemy poskładać sobie model do danej funkcjinalności a z Entity korzystać tylko w komunikacj i z bazą. np. niepotrzebne nam pole created/updated w Userze i mozna je spokojnie wywalić w Modelu który będziemy używać w danej funkcjonalności. Te modele w bazie mogą się zmieniać jeśli bedziemy chcieli rozwijać jakieś funkcjonalności więc imo wszędzie powinien zostać podział na Entity i Model